### PR TITLE
Instant reports sequence diagrams

### LINKED ISSUE #95: Prepare sequence diagram for Notification Service - instant reports
Prepare sequence diagram for Notification Service - instant reports

### PR DESCRIPTION
**Pull request by tisnik**
# Description

Instant reports sequence diagrams

Fixes #95

## Type of change

- Documentation update

## Testing steps

N/A

## Checklist
* [x] `make before_commit` passes
* [x] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: docs/instant_reports.puml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+activate "Notification\ndatabase"
+"ccx-notification-service" <-- "Notification\ndatabase": here are already reported result for cluster UUID
+deactivate "Notification\ndatabase"
+"ccx-notification-service" -> "ccx-notification-service" : find_differences(reported result, new result)
+activate "ccx-notification-service"
+"ccx-notification-service" <-- "ccx-notification-service": new rules
+deactivate "ccx-notification-service"
+"ccx-notification-service" -> "ccx-notification-service" : filter(new rules, total_risk_threshold)
+activate "ccx-notification-service"
+note over "ccx-notification-service" : currently any new rule\nwith any total risk is reported
```
Comment (epapbak) [inline docs/instant_reports.puml:31]: total risk higher than 3

Comment (tisnik) [reply docs/instant_reports.puml:31]: for instant messages? okok

Comment (epapbak) [reply docs/instant_reports.puml:31]: yep. Weekly digest is the one that doesn't filter at all =)

Comment (tisnik) [reply docs/instant_reports.puml]: fixed

**Review by epapbak — APPROVED** [review]
LGTM

**Pull request merged**